packages/css/src/components/Table/Table.stories.js

```diff
@@ -0,0 +1,90 @@
+export default {
+  title: 'Components/Table',
+  argTypes: {
+    stickyHeader: { control: 'boolean' },
+    disabled: { control: 'boolean' },
+    state: {
+      control: { type: 'select' },
+      options: ['default', 'disabled', 'active'],
+    },
+  },
+  decorators: [(Story) => `<div style="width: 450px;">${Story()}</div>`],
+};
+
+const Table = ({
+  stickyHeader = false,
+  state = 'default',
+  tableWidth = null,
```

Comment:
```suggestion
  tableWidth = '100%',
```

So that we don't have `null` here 
![image](https://user-images.githubusercontent.com/12156624/221162857-6db170fb-e4a5-4e22-b111-62ff8669e811.png)

Reply:
No, that's just for the example, so will make it optional to skip the `width: null`